spark-common/src/main/js/package.json

```diff
@@ -26,7 +26,8 @@
         "sass-loader": "^6.0.3",
         "string-replace-loader": "^1.2.0",
         "style-loader": "^0.16.1",
-        "webpack": "^2.3.3"
+        "webpack": "^2.3.3",
+        "uglifyjs-webpack-plugin": "1.0.0"
```

Comment:
The latest version of this plugin seems to be 1.2.7. Why was 1.0.0 chosen instead?